shared/psbt.py

```diff
@@ -1258,9 +1257,13 @@ def consider_inputs(self):
                 self.presigned_inputs.add(i)
 
             if not inp.has_utxo():
-                # maybe they didn't provide the UTXO
-                missing += 1
-                continue
+                if inp.subpaths and not inp.fully_signed:
```

Comment:
not sure why are you checking for presence of `inp.subpaths` when you try to decide if input is ours? You can create valid PSBT which has foreign subpaths but lacks foreign UTXO

this better:
`if inp.num_our_keys and not inp.fully_signed:`